app/controllers/api/v3/blood_pressures_controller.rb

```diff
@@ -26,6 +26,17 @@ def merge_if_valid(bp_params)
     end
   end
 
+  def after_merge_completed(results)
+    records = results.flat_map { |r| r[:record] }.compact
+    send_to_job(records)
+  end
+
+  def send_to_job(records)
```

Comment:
Naming nitpick. Can we pick a name that's more indicative of the "what" and not the "how"?

Reply:
Yup renamed. 👍